src/aspire/covariance/covar2d.py

```diff
@@ -281,6 +281,10 @@ def identity(x):
             A[k] = np.sqrt(weight) * ctf_fb_k_sq
             M += A[k]
 
+        logger.info("Check positive semidefinite property of left side b in Cov2D.")
+        if not b_coeff.check_psd_matrix():
+            logger.warning("Matrices are not positive semidefinite.")
```

Comment:
Which matrix? Be specific. (Same below.)

Reply:
Done in new commit 8d08e36.